src/v2/Apps/Consign/Components/GetPriceEstimate/ArtistInsightResult.tsx

```diff
@@ -0,0 +1,127 @@
+import React from "react"
+import { usePriceEstimateContext } from "./ConsignPriceEstimateContext"
+import { formatCentsToDollars } from "../../Utils/formatCentsToDollars"
+
+import {
+  Box,
+  Button,
+  Flex,
+  Image,
+  Separator,
+  Spacer,
+  Text,
+} from "@artsy/palette"
+import styled from "styled-components"
+
+export const ArtistInsightResult: React.FC = () => {
+  const { artistInsights, selectedSuggestion } = usePriceEstimateContext()
+
+  if (!artistInsights?.marketPriceInsights) {
+    return <ZeroState />
+  }
+
+  // TODO: Look into why we need to coerce these types from mp
+  const lowRangeCents: number = Number(
+    artistInsights.marketPriceInsights.lowRangeCents
+  )
+  const highRangeCents: number = Number(
+    artistInsights.marketPriceInsights.highRangeCents
+  )
+
+  const lowEstimateDollars = formatCentsToDollars(lowRangeCents)
+  const highEstimateDollars = formatCentsToDollars(highRangeCents)
+  const medianEstimateDollars = formatCentsToDollars(
```

Comment:
Just to check, this would only give the median if `lowRangeCents` and `highRangeCents` were the only estimate values we had. Is that right?

Reply:
@jonallured can confirm, but I _think_ `lowRangeCents` and `highRangeCents` already roll up the other values into a mean so these two values alone should be sufficient.

Reply:
A: `midRangeCents` is the field we're looking for